rel/config/runtime.exs

```diff
@@ -0,0 +1,85 @@
+use Mix.Config
+
+defmodule Environment do
+  @moduledoc """
+  This modules provides various helpers to handle environment metadata
+  """
+
+  def get(key), do: System.get_env(key)
+
+  def get_boolean(key) do
+    case get(key) do
+      "true" -> true
+      "1" -> true
+      _ -> false
+    end
+  end
+
+  def get_integer(key, default \\ 0) do
+    case get(key) do
+      value when is_bitstring(value) -> String.to_integer(value)
+      _ -> default
+    end
+  end
+
+  def exists?(key) do
+    case get(key) do
+      "" -> false
+      nil -> false
+      _ -> true
+    end
+  end
+
+  def version do
+    case Application.spec(:elixir_boilerplatecase, :vsn) do
+      nil -> Mix.Project.config()[:version]
+      version -> to_string(version)
+    end
+  end
+end
+
+# Configure application
+force_ssl = Environment.get_boolean("FORCE_SSL")
+scheme = if force_ssl, do: "https", else: "http"
+host = Environment.get("CANONICAL_HOST")
+port = Environment.get("PORT")
+
+config :elixir_boilerplate,
+  canonical_host: host,
+  force_ssl: force_ssl,
+  version: Environment.version()
```

Comment:
Since this doesn’t come from the runtime environment, we should put this in `config/config.exs` shouldn’t we?

And since the version in compiled at _build time_, we can safely use `Mix.Project.config()[:version]` 🎉  because `Application.spec(:elixir_boilerplatecase, :vsn)` is only available at runtime.

```elixir
# Configure application
config :elixir_boilerplate,
  ecto_repos: [ElixirBoilerplate.Repo],
  version: Mix.Project.config()[:version]
```

Reply:
Sure! No need for `Environment.version/0` anymore 👍